io_ogre/ui/export.py

```diff
@@ -235,6 +235,10 @@ def execute(self, context):
         name='Mesh Export Version',
         description='Specify Ogre version format to write',
         default=config.get('MESH_TOOL_EXPORT_VERSION'))
+    EX_XML_DELETE = BoolProperty(
```

Comment:
can you think of any scenario where the user wants to toggle this at *export time*?
I mean, In case someone wants to look at the XML, he can just specify an empty path for OGRETOOLS_XML_CONVERTER.

Reply:
Perhaps if something went wrong with the conversion from xml to binary the user might want to do a manual conversion and check the xml themselves.

Specifying an empty path for OGRETOOLS_XML_CONVERTER is annoying if you are testing and going back and forth.

Besides you might not know that blender2ogre will behave that way... someone might think that OGRETOOLS_XML_CONVERTER in mandatory and never think about it.

Besides I want the option to be visitble so people know that they can get the xml, someone might think that the xml is never generated somehow (I know this is a stretch, but its possible).

Mostly I want this to be somewhat visible, because the ability to see the xml can be important sometimes.

Reply:
what do you think about adding this prominently in the README instead?
Currently, this part of the install instructions is outdated anyway, so we could update it like:
> Set OGRETOOLS_XML_CONVERTER 
> - to OgreXMLConverter for ogre1
> - to OgreMeshTool for ogre-next
> - leave it blank to export to XML only (useful for debugging or external processing)

My objection is that while one intuitively thinks options make things more user friendly, the contrary is the case. Think of a new user who wants to get stuff into Ogre. He most likely cannot answer the question whether he needs that XML or not.
Compare that group to the group of people willing/ capable of debugging XML export issues. Who is more likely to read the README.

Reply:
I understand, but the reality is that most Ogre users are pretty advanced (yes, this contradicts some of my statements).

I think that both things should be done, updating the README and leaving the option.

The thing is that in my case I learned a lot by reading the description of the options, there were some features which I was unaware of.

And I think that Ogre users might want to know that the exporter is writing an xml file that can be inspected.

You know that most people don't read the README and go straight into action, leaving the option there helps to see that it is possible to inspect the output of the exporter.

On the other hand, new users are more likely to leave everything at default values so they won't see the xml being generated and discover that when they are more experienced and willing to tinker with the exporter options.

There is another scenario as well, suppose that someone is having trouble and we instruct them to post the xml... it is easier to say just uncheck this option.